tck-executor/start_pulsar.sh

```diff
@@ -2,7 +2,7 @@
 
 set -x -e
 
-IMAGENAME=apachepulsar/pulsar:2.8.1
+IMAGENAME=${PULSAR_IMAGE_NAME:-apachepulsar/pulsar:2.8.1}
```

Comment:
so you are going to call it this way ?
```
export PULSAR_IMAGE_NAME=xxxxx

mvn -Prun-tck ...
```

Reply:
yes, or 
`PULSAR_IMAGE_NAME=xxxx mvn -Prun-tck..`

the alternative was to pass a system property, read it from maven, pass it as argument to the script. the "problem" is that you would need to add arguments to different bash files, so I decided to go with the fastest way

let me know if you prefer the second option, is more "java"esque

Reply:
works for me

for reference....another way would have been to take the image version as parameter

`start_pulsar.sh apachepulsar/pulsar:2.8.1`
or
`start_pulsar.sh --image apachepulsar/pulsar:2.8.1
`